### DISCUSSION TITLE
Farming pools

### DISCUSSION THREAD

**Discussion by Gwhain** [Chia-Network/chia-blockchain#2965] [Q&A]
Hello everyone, 

does anyone have some info. about how to join a farming pool?
Last time I checked it was no possible yet, but it should be there at some point.
https://www.chiaexplorer.com/chia-farming-pools

Cheers
Gwhain

**Comment by dorkmo** [comment]
not yet, but i think its one of the top priorities right now

**Comment by oswaldzc** [comment]
you can try hpool.com or uupool.com
